general/server/users/adding-managing-users.md

```diff
@@ -0,0 +1,96 @@
+
+## Manage a User
+To manage a user either click on their portrait to go straight to their `Profile` tab, or click the `...` symbol inside that user's portrait. The later will open a small submenu with the options `Open` `Library access` `Parental control`, and `delete`. Except for delete, which does the obvious. These options will lead to different tabs but are otherwise all on the same page. `Open` corresponds to the Profile tab, `Library access` to access, `Parental control` to Parental Control, and there is an additional fourth tab `Password` for Password control. Changes to any option on any of these tabs need to be saved using `Save` at the bottom of the page.
+
+### Profile
+Directly under the tabs you have a link to `Edit this user's profile, image and personal preferences.` Clicking that allows you to change the user's personal settings, any setting here can be changed by both user and admin. <!--These options are described [on their own docs page](https://linktopage?) (not written yet)-->
+
+Under `name` you can change the user's name as it will be either displayed on, or has to be typed into login screen.
+
+Under `Authentication Provider` you have the option to change the backend that handles the login, by default the only option here will be `default` which means Jellyfin will handle this user, this option is sufficient for most use cases. Currently, the only other possibility is to have a LDAP server handle the login by installing the LDAP-Auth plugin. Note that if you wish to change a user's provider to LDAP after creating it in Jellyfin the username needs to be identical to the user's UID in LDAP, including capitalization.
+
+`Allow remote connections to this Jellyfin Server.` Unchecking this option will block login attempts this user makes from outside the networks defined as local, by default this will only be the subnet assigned to your network. But more can be added. <!-- More information about this can be found [on the docs page for `Expert` » `Advanced`](https://linktopage?) (not written yet)-->
+
+#### Feature Access
+For the following options it should be noted that if you never set up Live TV, users are blocked regardless of the state. See [the docs page for `Live TV` » `Live TV`](xref:server-live-tv-index) for more information.
+
+`Allow Live TV access` Unchecking this option will block the user's access to watch Live TV.
+
+`Allow Live TV recording management` Unchecking this option will block the user's access to set recording schedules.
+
+#### Media Playback
+`Allow media playback` Unchecking this option will block the user's access to media libraries, this does not include Live TV.
+
+`Allow audio/video playback that requires transcoding` Unchecking this option will block the user's access to video playback that requires transcoding (Decreases bandwidth use and quality)

```

Comment:
It's unclear which state (allowed or disallowed) decreases use and quality. Also it's probably worth linking at https://jellyfin.org/docs/general/server/transcoding.html somewhere here

Reply:
?

Reply:
Sorry, will add at some point, hopefully today